src/api.js

```diff
@@ -0,0 +1,22 @@
+const API_BASE_URL = "https://api.themoviedb.org/3/movie";
+const API_KEY = import.meta.env.VITE_APP_API_KEY;
+
+export const fetchPopularMovies = async () => {
+  const response = await fetch(
+    `${API_BASE_URL}/popular?api_key=${API_KEY}&language=en-US&page=1`
+  );
+  if (!response.ok) {
+    throw new Error("Failed to load popular movies");
+  }
```

Comment:
I only see this right now - maybe there's something wrong with the deployed env variable (VITE_APP_API_KEY)? 👀